content/doc/book/managing/cli.adoc

```diff
@@ -27,24 +27,18 @@ be convenient for scripting of routine tasks, bulk updates, troubleshooting,
 and more.
 
 The command line interface can be accessed over SSH or with the Jenkins CLI
-client, a `.jar` file distributed with Jenkins. The SSH approach is preferred
-over the CLI client as it is considered more secure.
+client, a `.jar` file distributed with Jenkins.
 
-////
-Background:
-23:19 <@rtyler> danielbeck: I'm writing CLI documentation, should I just encourage people to use SSH or stick with the default of jenkins-clir.jar?
-23:20 <@rtyler> we've told enough people to turn off the CLI over the past year...
-23:20 <@rtyler> :/
-23:21 <@danielbeck> it's probably better in general, but e.g. firewalls rules may require regular CLI use
-23:21 <@danielbeck> in any case, auth needs to use SSH keys
-////
+Prior to Jenkins 2.54, use of the custom CLI client is not recommended for security reasons:
```

Comment:
Right, it is the protocol, not the transport, which is the problem. Remoting protocol is insecure over either TCP or HTTP Duplex transport.

I will try to amend this text along the lines of what @rtyler suggests.